### PR TITLE
Add pinns utility for pinning namespaces

### PR DESCRIPTION
**Pull request by mrunalp**
@haircommander @umohnani8 ptal.

This is an initial just enough code to work version. I will clean it up.

### CODE REVIEW

File: pinns/pinns.c [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  }
+
+  if (bind_ipc) {
+    if (bind_ns(pin_path, "ipc") < 0) {
+      exit(1);
+    }
+  }
+
+  if (bind_net) {
+    if (bind_ns(pin_path, "net") < 0) {
```
Comment (haircommander) [inline pinns/pinns.c]: I find it a bit odd we exit 1 for failure but EXIT_SUCCESS upon succeeding

Comment (mrunalp) [reply pinns/pinns.c]: will be cleaned up :)

**Comment by haircommander** [comment]
/retest
LGTM

**Pull request merged**